### DISCUSSION TITLE
anything like SD upscale / MultiDiffusion  in StableSwarmUI?

### DISCUSSION THREAD

**Discussion by minsky91** [Stability-AI/StableSwarmUI#358] [Q&A]
Hello,
I tried StableSwarmUI recently (within Stability Matrix), it looks like a very promising platform. It's also very fast in rendering. I understand that it's still in early stages of development, with many features to be added later on, but when trying StableSwarmUI out I couldn't help but notice the absence of tiled upscaling in the package. I am doing a lot of high resolution upscaling lately, with target dimensions of up to 16K or higher, and a very fast img2img processing (on the par with Forge or better) is crucial for my project. Maybe I find SD upscale or MultuiDiffusion (or their equivalent) in some corner of UI that I didn't look into?

**Comment by CantainPike** [comment]
https://discord.gg/q2y38cqjNw
come to discord channel Discord is more active than here.

**Comment by mcmonkey4eva** [comment]
please post an [issue](https://github.com/Stability-AI/StableSwarmUI/issues) to track this as a feature request

I think comfy does some autotiling magic out of the box with large images, albeit not the same. You can also use Comfy nodes for this (albeit less convenient than a swarm native feature)
